src/class_sqlThread.py

```diff
@@ -358,16 +359,18 @@ def run(self):  # pylint: disable=too-many-locals, too-many-branches, too-many-s
             logger.debug('In messages.dat database, adding address field to the pubkeys table.')
             # We're going to have to calculate the address for each row in the pubkeys
             # table. Then we can take out the hash field.
-            self.cur.execute('''ALTER TABLE pubkeys ADD address text DEFAULT '' ''')
-            self.cur.execute('''SELECT hash, addressversion FROM pubkeys''')
-            queryResult = self.cur.fetchall()
-            from addresses import encodeAddress
-            for row in queryResult:
-                addressHash, addressVersion = row
-                address = encodeAddress(addressVersion, 1, hash)
-                item = '''UPDATE pubkeys SET address=? WHERE hash=?;'''
-                parameters = (address, addressHash)
-                self.cur.execute(item, parameters)
+            self.cur.execute('''ALTER TABLE pubkeys ADD address text DEFAULT '' ;''')
+
+            # create_function
```

Comment:
this should be a separate method/function, either in `helper_sql` or elsewhere in this file.